stm32-modules/include/thermocycler-gen2/thermocycler-gen2/motor_task.hpp

```diff
@@ -1009,13 +1098,13 @@ class MotorTask {
                 // Now that the lid is at the open position,
                 // the solenoid can be safely turned off
                 policy.lid_solenoid_disengage();
-                // Move to the Open Back To 90 step
+                // Overdrive into switch
                 policy.lid_stepper_start(
-                    LidStepperState::OPEN_BACK_TO_90_DEGREES, false);
+                    LidStepperState::OPEN_OVERDRIVE_DEGREES, true);
```

Comment:
this change is what removes the old open behavior of "always engage plate lift", right?

Reply:
Yep! Or sort of. On the older (1.4) prototypes, the limit switch is mounted far enough back that driving to it will put the lid at an angle much greater than 90º with the plate lift mechanism fully engaged. On the EVT units, the switch is at approximately 90º. So running the code before this PR on an EVT unit will result in the lid being open by like 70º and never activating the plate lift.